src/components/EventForm.js

```diff
@@ -6,21 +6,25 @@ import moment from "moment";
 import uuid from "uuid";
 
 export default class EventForm extends React.Component {
-  state = {
-    title: "",
-    description: "",
-    organisator: "",
-    city: "",
-    category: "",
-    image: false,
-    id: uuid(),
-    createdAt: moment(),
-    startDate: moment(),
-    startDateId: "",
-    endDate: moment(),
-    endDateId: "",
-    calendarFocused: null,
-  };
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: props.event ? props.event.title : "",
```

Comment:
Use destructuring for props `const { event } = props;` and thanks to that the block of code below would be a little bit smaller.